topics/icrc_29_window_post_message_transport.md

```diff
@@ -40,10 +39,16 @@ The message `targetOrigin` should be set to the previously received `icrc29_stat
 }
 ```
 
-> **Note**: The relying party should send `icrc29_status` messages in short intervals. It is expected that some of the messages will be lost due to being sent before the signer is ready.
+### Establishment
+
+The connection is considered established once the relying party receives a `"result": "ready"` response to an `icrc29_status` message, which may not necessarily be the first message sent. In case the relying party does receive any `"result": "ready"` response within a reasonable timeframe, it should treat this as a failure to establish the connection.
```

Comment:
- "which may not necessarily be the first message sent.": I'm not sure what that mean. Do you mean sent by the relying party or the signer?

- "In case the relying party does receive": isn't the keyword NOT missing here?

Reply:
Clarified why this may not be a response to the first message sent.

And yes, "not" was missing there.